pages/dkp/konvoy/2.1/release-notes/index.md

```diff
@@ -73,6 +73,36 @@ The following components have been upgraded to the listed version:
 - Node Feature Discovery 0.8.2
 - Nvidia Node Feature Discovery 0.4.1
 
+### Building on a CentOS OS using Pre-provisioned
```

Comment:
Is this an always and forever thing Matt? If yes, then we should find a section in the pre provisioned docs. If this is ONLY a thing that impacts 2.1 but no future version, then this is the right place. LMK your thinking @msdolbey